payjoin/src/uri.rs

```diff
@@ -332,6 +467,25 @@ mod tests {
                 assert!(Uri::try_from(&*uri).is_ok());
             }
         }
+
+        let base58 = "12c6DSiU4Rq3P4ZxziKxzrL5LmMBrzjrJX";
+        let bech32_upper = "TB1Q6D3A2W975YNY0ASUVD9A67NER4NKS58FF0Q8G4";
+        let bech32_lower = "tb1q6d3a2w975yny0asuvd9a67ner4nks58ff0q8g4";
+
+        for address in [base58, bech32_upper, bech32_lower].iter() {
```

Comment:
I think there's a more idiomatic way to do these iterators

```suggestion
        for address in vec![base58, bech32_upper, bech32_lower] {
```